introduction.md

```diff
@@ -0,0 +1 @@
+Hi Code School, I'm a good web developer and hope becoming a GREAT web developer thanks to your courses.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.